### PR TITLE
multiple potential talking points in initial investigations

### PR DESCRIPTION
**Pull request by jonathondilworth**
as requested.

### CODE REVIEW

File: The ADAO Project - Initial Investigations.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+#### 2. Minor Project Concerns
+
+There is another Cardano project with the name: ADADAO, it's listed as a legitimate stable-coin project on CardanoCube: [https://www.cardanocube.io/projects/adadao](https://www.cardanocube.io/projects/adadao).
+
+The information on their website talks about tokens such as AUSD and ADAO (this could be confusing for people as they recognise both this project and ADADAO).
+
+#### 2.1. Considerations: Minor Project Concerns
+
+* Is this something we should be taking into consideration?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:20]: Yes it definitely is an issue, our website was launched prior to theirs so it all boils down to trademarking ADAO. TCT will look into this. Potentially trademarking it via his own company.

File: The ADAO Project - Initial Investigations.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Right, so I've only really been reading through all the ADAO channels, tweets, the website, etc for the last day or two. However, I really do like the project concept & I would be very happy to get more involved.
+
+#### 3.1. Project Related Questions
+
+After having briefly reviewed the above, I have some questions (which may or may not have answers - I may have simply not come across them, if this is the case, my apologies. I am doing my best to get up to speed as fast as possible).
+
+#### 3.1.1. Community Engagement
+
+* What strategies are we currently utilising to generate community engagement with ADAO?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:34]: - Two weekly meetings (Saturday 8pm UTC and Thursday 2am UTC) 
- Twitter Spaces
- Breakout Sessions 
- Events hosted on our server  where we welcome guests from other projects 

There is no definitive strategy yet. We also thought about having weekly Twitter spaces

File: The ADAO Project - Initial Investigations.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+* Have we exhaustively considered as many strategies as possible?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:35]: We have discussed a lot but as many projects we lack documentation.

File: The ADAO Project - Initial Investigations.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+* Do we have individuals who are solely responsible for developing community engagement?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:36]: No, might be something we need to consider

File: The ADAO Project - Initial Investigations.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+#### 3.1.2. Brand Awareness
+
+* Similarly, what are our strategies surrounding raising brand awareness?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:40]: -  Self hosted Twitter spaces
- Blog posts (?) _That's still under development_

There is no definitive strategy yet defined

File: The ADAO Project - Initial Investigations.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+* Does anybody feel that alternative strategies may be more fruitful in terms of accomplishing a higher degree of brand awareness?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:41]: What alternative strategies are you talking about?
I feel like we should definitely get in touch with more projects that plan to establish DAOs as they would be our target audience

File: The ADAO Project - Initial Investigations.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+* Do we have the resources necessary to implement increasingly successful strategies to raise brand awareness (such as marketing professionals)?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:42]: As far as I'm aware we do have some seasoned marketeers among our ranks. 
I know that Ibis has a lot of experience.
There is even a server role called Marketeers that you could reach out to

File: The ADAO Project - Initial Investigations.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+#### 3.1.3. Content Creation
+
+* What kind of content do we currently produce, other than posts on Twitter and posts within this discord channel?
+* Do we record meetings and post them online for transparency, or do we simply take minutes?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:47]: Not yet. we only take minutes.
We could think about recording the audio is our meetings are most of the time only audio based

File: The ADAO Project - Initial Investigations.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+
+#### 3.1.2. Brand Awareness
+
+* Similarly, what are our strategies surrounding raising brand awareness?
+* Does anybody feel that alternative strategies may be more fruitful in terms of accomplishing a higher degree of brand awareness?
+* Do we have the resources necessary to implement increasingly successful strategies to raise brand awareness (such as marketing professionals)?
+
+#### 3.1.3. Content Creation
+
+* What kind of content do we currently produce, other than posts on Twitter and posts within this discord channel?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:46]: - Meeting Minutes in our GitHub Repo

Besides that we don't produce content yet.
Blog is planned.
One Video was created from our Idea Fest presentation https://drive.google.com/file/d/1936r_56iMCcMoXpSedB3tceFG4PSaDtd/view?usp=drivesdk

File: The ADAO Project - Initial Investigations.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+* Do we record meetings and post them online for transparency, or do we simply take minutes?
+* What do our ADAO members think about potentially recording meetings?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:48]: Okay for me. Maybe we should ask in the server. 
Due to GDPR we have to ask for consent before we start recording

File: The ADAO Project - Initial Investigations.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+* Do we have an available blog?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:49]: Not yet.
Working group has been established

File: The ADAO Project - Initial Investigations.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+* Do we have dedicated content creators?
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:50]: There is server role called Marketeers that you could reach out to.
I know that Nate @Nastradamus created some content.

File: The ADAO Project - Initial Investigations.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+
+#### 4. Potential Short Term Contributions
+
+Within this section I will outline some areas of work that I believe I can be instrumental or at least helpful in developing for ADAO.
+
+#### 4.1. An Initial 'Naive' Marketing Strategy
+
+Having ran a small business in the past, I am familiar with a variety of marketing tools. I would suggest the following:
+
+* Building a set of newsletter audiences and serving newsletters via MailChimp (I'm happy to finance this if other ADAO members are committed to helping out in terms of content creation and newsletter creation).
```
Comment (thenic95) [inline The ADAO Project - Initial Investigations.md:77]: That sounds brilliant. I would love to contribute. There is an already existing working-group. We could use that channel for further discussions.

File: The ADAO Project - Initial Investigations.md (hunk 9 above)
Comment (zachyking) [reply The ADAO Project - Initial Investigations.md:46]: We should have a youtube channel, quite a bit of recording were already posted in just a few weeks I am in Discord. This could be used to boost the branding more if we have logos everywhere and a unified design style on videos. And youtube can get its own reach.

File: The ADAO Project - Initial Investigations.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+* What kind of content do we currently produce, other than posts on Twitter and posts within this discord channel?
+* Do we record meetings and post them online for transparency, or do we simply take minutes?
+* What do our ADAO members think about potentially recording meetings?
+* Do we have an available blog?
+* Do we have dedicated content creators?
+* What kind of content do ADAO members think would be most appropriate to produce in order to achieve better community engagement and a larger degree of brand awareness?
+
+#### 3.1.4. Educational Information
+
+* Is educational information related to DAOs something we may consider providing to a general audience and the community at large?
```
Comment (zachyking) [inline The ADAO Project - Initial Investigations.md:55]: I think this is something going on (intro-to-daos discord channel, and twitter spaces discussions) but some content and sources could be promoted on our own blog or webpage.

And the general consensus is definitely to try providing education to the broader public.

File: The ADAO Project - Initial Investigations.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+
+* Do we have any plans to produce new graphics related to ADAOs branding? e.g. Project Infographics & related industry Infographics?
+* Do we have a branding bible for the project?
+
+#### 3.1.6. The ADAO Website
+
+* What is the general consensus on ADAOs current website and its form? Is a very simple one page website appropriate for a project of this magnitude?
+* What do ADAO members think of the current ADAO website?
+* Is the website hosted on IPFS (as it is a single page, but I noticed there were JS scripts which made it look like some kind of WordPress install, which I thought was strange, there is also quite a bit of unnecessary CSS and JS that is loaded on each page request).
+* If ADAO had a web developer work on building out a much more informative website, would this be something that worth spending some time on?
```
Comment (zachyking) [inline The ADAO Project - Initial Investigations.md:67]: I know you already got some response to this from Nate, I think most of us devs in ADAO, including myself, are not too UX/UI oriented but for sure we can help to build. 

At least a little bit of educational and promotional content on a nice website with all the links could have an impact.

File: The ADAO Project - Initial Investigations.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+* Using as any avenues of social media as possible, but primarily targeting building a larger following on Twitter, Facebook and Instagram. I am friends with an extremely successful marketing expert for whom I can have a chat with and create some strategy documents for accomplishing agreed upon goals.
+* I would be happy to implement some strategies so long as help from other ADAO members was provided in order to maintain these strategies over time.
+
+#### 4.2. Methodologies, Interfaces & Incentive Structures For Content Creation & Educational Resources
+
+Personally, I believe educational resources are key for adoption of these novel technologies which individuals outside of our industry / domain really don't understand. Thus, I would be happy to either install an already existing blogging platform (such as WordPress) or even build out BASIC bespoke system (which I have done before in a previous job - it was a CRUD system which created 'articles' to help graduates find jobs and to give them advice on doing so).
```
Comment (zachyking) [inline The ADAO Project - Initial Investigations.md:83]: I would probably prefer something like Next.js that can use markdown files to generate content, either dynamically or statically on build really easily.
We can get easy setup free preview deployment on each commit with Vercel and use git to approve and deploy new articles in at least close to a decentralized manner (set number of approvals).

Again, I am not the best for UI/UX, but happy to set this up potentially.

Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:83]: I would tend to agree, given the nature of our members as of current. I have built fairly large websites on top of Hugo which is a static site generator that uses YAML files and a bunch of MarkDown files; so I am familiar with this sort of technology. So long as users understand how to write MarkDown, I see no issue with this.

I know a UX student that may be worth consulting, as I am not academically trained in UX myself, but she does UX research, so she may have some additional ideas or critiques.

File: The ADAO Project - Initial Investigations.md (hunk 15 above)
Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:67]: Personally, I think educational content is pretty key (and is in demand!). I always have to step back for a moment when speaking to somebody who does not reside within our industry because I have to remind myself that: this person likely doesn't truly understand what an NFT is, or what a DAO (if they've even heard of a DAO) is, or even the difference between something like ETH and BTC. Most people, I've found, only really know about ETH, BTC or Dogecoin (every time somebody asks me about a meme coin, my faith in humanity starts to dwindle - I have to remind myself to be compassionate towards others).

File: The ADAO Project - Initial Investigations.md (hunk 14 above)
Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:55]: Indeed, I am thinking of asking a good friend of mine, an animator, to see if he would be interested in producing some animation videos that explain concepts directly related to our project and similar projects, I do think that may provide us with an edge as a project since we have made the claim to aim to provide a framework **and resources** to make it easier for others to operate their own DAOs, right?

File: The ADAO Project - Initial Investigations.md (hunk 13 above)
Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:77]: Yeah, I will be engaging with that workgroup shortly :) I have Mailchimp setup already, we'll just need to register appropriate e-mail co-respondents and information relating to GDPR, etc. Thus, we can expect to see some progress here very soon.

File: The ADAO Project - Initial Investigations.md (hunk 11 above)
Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:49]: Okay, I'll start engaging with that soon.

File: The ADAO Project - Initial Investigations.md (hunk 5 above)
Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:40]: Okay, so essentially we just need a formalised marketing strategy, branding bible, etc. It may be worth considering putting together a business model canvas? It addresses a lot of these issues, even though a DAO is (obviously) not a conventional business.

File: The ADAO Project - Initial Investigations.md (hunk 1 above)
Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:20]: I will leave this in the project owners capable hands! Let me know if any assistance from myself is required or if there is anything I can do.

File: The ADAO Project - Initial Investigations.md (hunk 16 above)
Comment (zachyking) [reply The ADAO Project - Initial Investigations.md:83]: Writing markdown should not be a problem, we can definitely assist someone submitting PR, and there is also https://word2md.com/.

I think the biggest advantage there would be functional decentralization granted for free when using git :) 

At least a feedback from UX person would be definitely welcome!

File: The ADAO Project - Initial Investigations.md (hunk 5 above)
Comment (thenic95) [reply The ADAO Project - Initial Investigations.md:40]: I'm down.
We could do a live session filling out the business model canvas together.

Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:40]: Yeah, I would agree insofar as that is the best way to populate a business model canvas. Every time I've done one before, we would ensure that everybody was present. It's quite easy to miss details out if you don't have at least one person from each major business 'role' sitting there with you. Do we have a Trello, or... what project management tools are we using, just GitHub? I think I saw a KanBan board somewhere, but the last day or two has been pretty patchy, the worst Christmas ever lol. If we have a couple of individuals willing to do a session, that should be fine, so I'll start looking to organise that for a date that suits everybody.

Typically you produce multiple canvases at different levels of 'business' granularity, right? I'll need to whip out New Venture Creation and re-read up on some of this stuff, it's been a while since I last took part in a serious project/start-up!

TLDR: I'll get on organising this ASAP.

Comment (thenic95) [reply The ADAO Project - Initial Investigations.md:40]: Here is the link to our trello board:

https://trello.com/b/FaUudFcZ/main-board

It's not used as much as I would like it to be.. 
Somehow it's really hard to get members to use tool like trello or github. Especially if they haven't used it before.
I think we should have dedicated work group leaders/PMs that take care of organizing everything.
I try to do it myself but on the other hand I also want to actively participate and then i don't have enough time to do anything the way I would like to.

Don't worry about organizing that. First you should focus on getting better!

And your absolutely right input from various stakeholders even from inside our community is an absolut must.

Comment (jonathondilworth) [reply The ADAO Project - Initial Investigations.md:40]: I've gotten over the worst of it now, it's like day six today, so I should be absolutely fine within the next three days or so. It's going to be a COVID Christmas for me this year haha. I don't have much else to do, so I figured I might as well contribute and do something when I'm feeling up to it. Discord is great, it's pretty much replaced Slack, Git or GitHub is kind of required for software dev (obviously) and then anything from Atlassian is pretty good as well - when you are actually getting stuff done, Trello is super useful IMO. You can't meaningfully contribute within a team without these tools, otherwise you'll get nowhere slowly lmao.

Is there a way of automatically producing a calendar .ics file from an organised Discord event? That's something that I would find useful, but I haven't yet come across a built-in means to do so.

**Comment by thenic95** [comment]
I like the way you wrote the meeting minutes.
Very informative and concise!
